app/scripts/views/settings.js

```diff
@@ -148,6 +149,34 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
       return self._showAvatar();
     },
 
+    beforeDestroy: function () {
+      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
+        $('body').removeClass('settings').show();
+      });
+    },
```

Comment:
> Yeah, that transition messed with the removal of the layoutClassName, so I took it out.

I now see not having it messes with other things. In base.js, can you document why the transition is needed or else another drive by removal could occur.